prov/efa/src/rxr/rxr_env.c

```diff
@@ -90,7 +91,7 @@ void rxr_env_param_get(void)
 		FI_WARN(&rxr_prov, FI_LOG_EP_DATA,
 			"FI_EFA_TX_MIN_CREDITS was set to %d, which is <= 0."
 			"This value will cause EFA communication to deadlock."
-			"To avoid that, the variable was reset to %d\n",
+			"To avoid that, the variable was reset to %dn",
```

Comment:
Same here, looks like you may have accidentally stripped some backslashes fron the source code.

Reply:
OBE.